user-templates-api-spec.yaml

```diff
@@ -0,0 +1,151 @@
+openapi: 3.0.0
+info:
+  description: The User Templates API returns pre-defined templates to be used by interactive services (such as JupyterLab).
```

Comment:
This is very minor but maybe title should be the first element in the info object?